pulsar-broker-common/src/main/java/com/yahoo/pulsar/broker/ServiceConfiguration.java

```diff
@@ -62,6 +62,7 @@
     private long zooKeeperSessionTimeoutMillis = 30000;
     // Time to wait for broker graceful shutdown. After this time elapses, the
     // process will be killed
+    @FieldContext(dynamic = true)
```

Comment:
I'd leave this PR to implement the "mechanism" and then we can make the actual variables dynamic in subsequent PRs.